features/step_definitions/salt_steps.rb

```diff
@@ -864,9 +864,10 @@
   end
 end
 
-Then(/^I run spacecmd listevents for sle-minion$/) do
+Then(/^I run spacecmd listevents for ([^\s]*)$/) do |minion|
+  fullhostname = get_target_fullhostname(minion)
```

Comment:
in sake of completeness, can we call the var minion_fullhostname :panda_face: ,  since it could be a trad_client